### ISSUE TITLE
wireframe model is irrelevant

### ISSUE THREAD

**Issue by mhashas** [DQiaole/ZITS_inpainting#41]
Hi, 

I've been playing quite a bit with your model due to the amazing results. Something that i've noticed is that the wireframe model is irrelevant. If I return a full zeros tensor of `lines_tensor` of the same shape as the actual output in `wf_inference_test`, I get the same final outputs. Is there a bug somewhere?

To replicate:

`return torch.zeros_like(lines_tensor.detach())` in `wf_inference_test`

Update:

It seems that also the edges seem to be useless. 
```
  batch["line_256"] = torch.zeros_like(batch["mask_256"])
  batch["line"] = torch.zeros_like(batch["mask_512"]) 
  batch["edge"] = torch.zeros_like(batch["mask_512"])
```
Do this change gives me the same results

Let me know if im doing something wrong

**Comment by DQiaole** [comment]
Hi~
I think this is determined by the content of your images, if the image contains only some simple edges and wireframes, then it may not need to provide a explicit structural input to FTR, which can also complete the inpainting task alone. For more complex images, such as the indoor dataset we provide, the structure input indeed provide additional information. 
On the other hand, since our TSR sometimes generates some "incorrect" structural information, this may also lead to our subsequent FTR selectively ignoring structural information to bring more robust image inpainting results during training.  It may also account for this phenomenon.

**Comment by mhashas** [comment]
Hi,

Thanks for the answer. I will test with multiple images and update my answer for the other ones. Thanks again for open-sourcing this model, great performance!

**Comment by mhashas** [comment]
On closer inspection, there are cases in which the outputs vary for the human eye, albeit by not a large margin. By analyzing pixel values, outputs do vary in all cases, although in most they are indistinguishable. If you need a speed-up, removing those parts achieves this with a minimal performance hit. 

Thanks again
